content/kubermatic/master/installation/install_kubermatic/_index.en.md

```diff
@@ -18,3 +18,370 @@ At the moment you need to be invited to get access to Kubermatic's Docker regist
 * **Master cluster** -- A Kubernetes cluster which is responsible for storing the information about users, projects and SSH keys. It hosts the Kubermatic components and might also act as a seed cluster.
 * **Seed datacenter** -- A definition/reference to a seed cluster
 * **Node datacenter** -- A definition/reference of a datacenter/region/zone at a cloud provider (aws=zone, digitalocean=region, openstack=zone)
+
+## Requirements
+
+Before installing, make sure your Kubernetes cluster meets the [minimal requirements]({{< ref "../../requirements" >}})
+and make yourself familiar with the requirements for your chosen cloud provider.
+
+For this guide you will have to have `kubectl` and [Helm](https://www.helm.sh/) (version 2 or 3) installed locally.
+
+## Installation
+
+To begin the installation, make sure you have a kubeconfig at hand, with a user context that grants `cluster-admin`
+permissions.
+
+### Clone the Installer
+
+Clone the [installer repository](https://github.com/kubermatic/kubermatic-installer) to your disk and make sure to
+checkout the appropriate release branch (`release/vX.Y`). The latest stable release is already the default branch,
+so in most cases there should be no need to switch. Alternatively you can also download a ZIP version from GitHub.
+
+```bash
+git clone https://github.com/kubermatic/kubermatic-installer
+cd kubermatic-installer
+```
+
+### Create a StorageClass
+
+Kubermatic uses a custom storage class for the volumes created for user clusters. This class, `kubermatic-fast`, needs
+to be manually created during the installation and its parameters depend highly on the environment where Kubermatic is
+installed.
+
+It's highly recommended to use SSD-based volumes, as etcd is very sensitive to slow disk I/O. If your cluster already
+provides a default SSD-based storage class, you can simply copy and re-create it as `kubermatic-fast`. For a cluster
+running on AWS, an example class could look like this:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: kubermatic-fast
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+```
+
+Store the above YAML snippet in a file and then apply it using `kubectl`:
+
+```bash
+kubectl apply -f aws-storageclass.yaml
+```
+
+Please consult the [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#parameters)
+for more information about the possible parameters for your storage backend.
+
+### Install Helm's Tiller
+
+{{% notice note %}}
+This step is only required when using Helm 2.
```

Comment:
Should we add that we recommend Helm 3?

Reply:
Do we? All our installation instructions are still Helm 2-based and we don't run Helm 3 in production anywhere yet.

Reply:
Some TNG guys tried helm3 and it worked, but they needed to do some commands different - In my opinion we should expose this script to the customers and check the helm version: https://github.com/kubermatic/kubermatic/blob/master/api/hack/deploy.sh. A script to use all "not operator driven parts" would be very helpful for customers.